app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java

```diff
@@ -264,6 +265,7 @@ private void setupViewPager( ViewPager viewPager )
 		{
 			adapterResult.addFragment( new NutritionProductFragment(), menuTitles[2] );
 			adapterResult.addFragment( new NutritionInfoProductFragment(), menuTitles[3] );
+			adapterResult.addFragment( new EnvironmentProductFragment(), "Environment" );
```

Comment:
You need to add this to `menuTitles` string array and then reference it here. Do the same for other entries that may have missed this logic.

Reply:
Do I need to add it manually to all the languages? Adding it to the default strings.xml doesn't seem to work

Reply:
@KarthikRIyer No, you don't need to add it manually to all the languages, just add the String to the default `string.xml` file. The translations are handled by Crowdin. You don't need to worry about that.